### ISSUE TITLE
Masking demo looks weird.

### ISSUE THREAD

**Issue by 3vilguy** [pixijs/examples#14]
There's something wrong with [Masking demo](http://pixijs.github.io/examples/index.html?s=demos&f=masking.js&title=Masking).

Maybe it's intentional but I don't think so. It [used to look](http://www.goodboydigital.com/pixi-js-brings-canvas-and-webgl-masking/) like this:

![old](https://cloud.githubusercontent.com/assets/2122619/15143207/75d8ee1a-16a0-11e6-8348-fadfc1666883.png)

And now it's like:

![current](https://cloud.githubusercontent.com/assets/2122619/15143221/8a8c4b90-16a0-11e6-8c80-2d6d634488df.png)

After bit of digging looks like [this code](https://github.com/pixijs/examples/blob/gh-pages/examples/demos/masking.js#L98) was shorted before:

```
thing.beginFill(0x8bc5ff, 0.4);
thing.moveTo(-120 + Math.sin(count) * 20, -100 + Math.cos(count)* 20);
thing.lineTo(120 + Math.cos(count) * 20, -100 + Math.sin(count)* 20);
thing.lineTo(120 + Math.sin(count) * 20, 100 + Math.cos(count)* 20);
thing.lineTo(-120 + Math.cos(count)* 20, 100 + Math.sin(count)* 20);
thing.lineTo(-120 + Math.sin(count) * 20, -100 + Math.cos(count)* 20);
thing.rotation = count * 0.1;
```

So I'm guessing something went wrong while moving examples around.
If somehow it's how it's suppose to look like, just close this issue.

**Comment by bigtimebuddy** [comment]
Fixed with #24. Thanks for pointing this out.

**Comment by 3vilguy** [comment]
No problem 👍
Thanks for fix :)
